tests/test_api.py

```diff
@@ -57,6 +57,10 @@ def test_non_deterministic_QuerySet_annotate(self):
         with record():
             list(Author.objects.annotate(x=Upper("name"), y=Upper("name")))
 
+    def test_dependent_QuerySet_annotate(self):
+        with record():
+            list(Author.objects.annotate(y=Upper("name"), x=F("y")))
```

Comment:
Given that the captured snapshot for this seems to elide the selected values, would a better test to use the kwargs somewhere that they will appear in the snapshot? From a quick look at the other tests (e.g: `test_non_deterministic_Q_query`), perhaps using them to filter might achieve that?

Reply:
Done in #539, thanks for the suggestion! How on earth did you spot this in a 2 year old PR?

Reply:
I ended up having a look at this PR when #12 was closed :) (I hadn't noticed that the PR was that old!)